cli/modules/signup/index.js

```diff
@@ -0,0 +1,69 @@
+const prompt = require('prompt')
+const { requestApi } = require('../../utils')
+
+main()
+
+async function main () {
+  const { email, username } = await promptUserInformation()
+  const password = await promptPassword()
+  const { error } = await requestSignup(email, username, password)
+  if (error) throw new Error(error)
+  console.log('Signed up successfully. You can now login')
+}
+
+async function requestSignup (email, username, password) {
+  const res = await requestApi('auth/signup', {
+    method: 'POST',
+    body: JSON.stringify({
+      email,
+      username,
+      password
+    }),
+    headers: {
+      'Content-Type': 'application/json'
+    }
```

Comment:
We need to manually add the `Content-Type` header when sending JSON payloads using the fetch API.